primitives/src/v1.rs

```diff
@@ -168,10 +160,10 @@ impl<H> CandidateReceipt<H> {
 pub struct FullCandidateReceipt<H = Hash, N = BlockNumber> {
 	/// The inner candidate receipt.
 	pub inner: CandidateReceipt<H>,
-	/// The global validation schedule.
-	pub global_validation: GlobalValidationData<N>,
-	/// The local validation data.
-	pub local_validation: LocalValidationData<N>,
+	/// The validation data derived from the relay-chain state at that
+	/// point. Should match the `validation_data_hash` in the descriptor
+	/// of the receipt.
+	pub validation_data: ValidationData<N>,
```

Comment:
Again, the hash of the whole struct should match the one in the descriptor or only the hash of `persisted` field?